### DISCUSSION TITLE
Does FreeTube personalise videos?

### DISCUSSION THREAD

**Discussion by Raj2032** [FreeTubeApp/FreeTube#2616] [General]
Hi, I would love to move away from YouTube to FreeTube and use it as a replacement, however I was wondering if FreeTube, like YOuTube personalises the videos on my feed based on the videos I watched and what I am subscribed to?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No, FT does not personalise videos. For more info see https://docs.freetubeapp.io/usage/privacy/#tracking
